File: isPalindrome.js
// function isPalindrome(x) {
  // x = x.toLowerCase()
  // console.log(x);
  // x = x.split('');
  // console.log(x);
  // x = x.reverse()
  // console.log(x);
  // x = x.toString()
  // console.log(x);

function isPalindrome(x) {
  x = x.toLowerCase()
  console.log(x.split('').reverse().join(''));
  if (x === x.split('').reverse().join('')) {
    console.log(true);
  } else {
    console.log(false);
  }
}

isPalindrome('aba');
isPalindrome('Aba');

File: is_palindrome.rb
# def is_palindrome str
#   str = str.downcase
#   puts str
#   # print str.split("").reverse.join("")
#   if str == str.split("").reverse.join("")
#     puts true
#   else
#     puts false
#   end
# end

def is_palindrome str
  str = str.downcase
  puts str
  # print str.split("").reverse.join("")
  if str == str.reverse
    puts true
  else
    puts false
  end
end

is_palindrome("abba")
is_palindrome("Abba")
is_palindrome("hello")
